Servers/Atlas/.arc/submit.py

```diff
@@ -97,7 +104,17 @@
 
 touch initial_time
 
-/Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 input.in
+mkdir -p $MOLPRO_SCRDIR
+cd $MOLPRO_SCRDIR
+
+cp "$SUBMIT_DIR/input.in" .
+
+/Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 output.out
```

Comment:
Minor fix up:
It should be this

`/Local/ce_dana/molpro-mpp-2022.2.3/bin/molpro -n {cpus} -t 1 input.in -o output.out`